src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,102 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import mockVideos from '../../mock/youtube-videos-mock.json';
+import Switch from '../../components/Switch';
+// import { useAuth } from '../../providers/Auth';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import { NavBar, Container, VideoCard } from './Home.styled';
 
 function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+  // const history = useHistory();
+  // const { authenticated, logout } = useAuth();
+  const [active, setActive] = useState(false);
```

Comment:
maybe this state will change to context or a global style because other pages will use it too